src/AsyncStatusIndicator/AsyncStatusIndicator.tsx

```diff
@@ -27,16 +52,16 @@ export const Props = {
  */
 @skinnable()
 @props(Props)
-export default class AsyncStatusIndicator extends React.PureComponent {
+export default class AsyncStatusIndicator extends React.PureComponent<AsyncStatusIndicatorProps, void> {
```

Comment:
(probably not anymore in 16 :D https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L269)